src/parsers/perf/perfparser.cpp

```diff
@@ -503,6 +503,33 @@ struct SymbolData
     }
 };
 
+struct CallerCalleeLocation
+{
+    QString symbol;
+    QString binary;
+
+    bool operator<(const CallerCalleeLocation& location) const
```

Comment:
This is consistent with your coding style in heaptrack, for example: bool operator<(const LocationData::Ptr& rhs) const

in analyze/gui/parser.cpp

And in Qt code for example: bool operator<(const Module& module) const

in perfsymboltable.cpp

or am I misunderstanding what you mean here?